xeus-zmqConfig.cmake.in

```diff
@@ -22,7 +22,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR};${CMAKE_MODULE_PATH}")
 @XEUS_ZMQ_CONFIG_CODE@
 
 include(CMakeFindDependencyMacro)
-find_dependency(nlohmann_json @nlohmann_json_REQUIRED_VERSION@)
+find_dependency(nlohmann_json @nlohmann_json_REQUIRED_VERSION@ EXACT)
```

Comment:
This should be fixed as in [xeus](https://github.com/jupyter-xeus/xeus/commit/6e95bb94409d7b86bbf0a5a36dac0f7c4335ccd7)